.changeset/angry-cheetahs-march.md

```diff
@@ -0,0 +1,5 @@
+---
+'@emotion/native': patch
+---
+
+updating @types/react-native and react-native
```

Comment:
Match supported components to what is exported from the latest version of React Native (0.61.5).